### ISSUE TITLE
VSCodium has removed mangling; update to support latest version

### ISSUE THREAD

**Issue by battaglia01** [iocave/customize-ui#162]
VSCodium has removed the mangling and thus restored CustomizeUI functionality. However, CustomizeUI needs to be updated to be usable in the latest version of VSCodium, as some file it is looking for is missing.

However in general the VSCodium devs have expressed they are removing the mangling deliberately for the benefit of plugins like CustomizeUI. See here: https://github.com/VSCodium/vscodium/issues/1390#issuecomment-1407788309

**Comment by lehni** [comment]
It's actually https://github.com/iocave/monkey-patch that needs the update to work. The current CustomizeUI works fine on VSCodium once you install a custom-built monkey-patch: https://github.com/VSCodium/vscodium/issues/1390#issuecomment-1411187403

**Comment by knopp** [comment]
I lost track of all this a bit. Where there JS loader changes in vscodium that broke monkey patch?

**Comment by lehni** [comment]
@knopp yes, basically we'd need a release with this rolled in: https://github.com/iocave/monkey-patch/commit/065d2ea88d76ec5110103058631bcdfe620161cb

**Comment by rminami** [comment]
Which version should this be fixed on? I'm currently running

- VSCodium v1.76.2
- Monkey Patch v0.1.23
- Customize UI v0.1.65
- macOS 13.2.1

but running "Enable Monkey Patch" seems to break the installation (the window doesn't reopen and the only menu bar available is "Quit"). 

I tried installing VSCodium Insiders (v1.77.0) as well, but I get the same thing.

**Comment by lehni** [comment]
@rminami you need to use a custom built version of monkey patch with the latest changes rolled in:

https://github.com/VSCodium/vscodium/issues/1390#issuecomment-1411187403
https://github.com/iocave/monkey-patch/commit/065d2ea88d76ec5110103058631bcdfe620161cb
